src/rvsdg/egglog_optimizer/schema.egg

```diff
@@ -63,7 +63,7 @@
 (function Gamma (Operand VecOperand VecVecOperand) Body)
 ;; loops
 ;; predicate (inside loop), inputs (outside loop), outputs (inside loop)
-(function Theta (Operand VecOperand VecOperand) Body)
+(function Theta (Operand VecOperand VecOperand) Body :cost 20)
```

Comment:
Do other nodes have specific costs? Is the number 20 important? I can sort of guess what this does but a comment here would be good.

Reply:
Oops, I didn't mean to keep this change. I removed it, good catch.
In the future we will need a better cost function but not needed for this PR.